sygnal/apnspushkin.py

```diff
@@ -132,7 +132,11 @@ def __init__(self, name, sygnal, config):
                 raise PushkinSetupException("You must supply topic.")
 
         if certfile is not None:
-            self.apns_client = APNs(client_cert=certfile, use_sandbox=self.use_sandbox)
+            self.apns_client = APNs(
+                client_cert=certfile,
+                use_sandbox=self.use_sandbox,
+                max_connection_attempts=0,
```

Comment:
I'm guessing that this parameter really means "max *additional* connection attempts if the first fails"?

This might need a comment saying that there's a number of connection attempts, but also a number of retries that occur?

Reply:
Yes, could do with a comment :)